src/runtime/managedtype.cs

```diff
@@ -29,7 +29,7 @@ internal static ManagedType GetManagedObject(IntPtr ob)
                 }
 
                 var flags = Util.ReadCLong(tp, TypeOffset.tp_flags);
-                if ((flags & TypeFlags.Managed) != 0)
+                if ((flags & (int)TypeFlags.Managed) != 0)
```

Comment:
Same, `HasFlag`

Reply:
@koubaa This is not really resolved, is it?